packages/jellyfish-network/src/BlockRewardDistribution.ts

```diff
@@ -0,0 +1,50 @@
+import BigNumber from 'bignumber.js'
+
+export interface BlockRewardDistribution {
+  masternode: number
+  community: number
+  anchor: number
+  liquidity: number
+  loan: number
+  options: number
+  unallocated: number
+}
+
+export const BlockRewardDistributionPercentage: BlockRewardDistribution = {
```

Comment:
`export const BlockRewardDistributionPercentage: () => BlockRewardDistribution`

perhaps make it slightly easier for future upgrade, let say split percentage change after certain height.

Reply:
Hmmm, that's true. 

Perhaps an issue for later? Not sure how to design it now since the change is unclear.